Makefile

```diff
@@ -112,7 +116,33 @@ ifeq ($(TARGET_ARCH),arm64)
 endif
 endif
 
-.PHONY: build-ffmpeg tool-install gofmt lint test profile-cpu profile-memory update-rdk module clean clean-all
+ifeq ($(TARGET_OS),windows)
+ifeq ($(SOURCE_OS),linux)
+ifeq ($(TARGET_ARCH),amd64)
+    GO_TAGS ?= -tags no_cgo
+    X264_ROOT ?= $(shell pwd)/x264/windows-amd64
+    X264_BUILD_DIR ?= $(X264_ROOT)/build
+    # We need the go build command to think it's in cgo mode
+    export CGO_ENABLED = 1
+    # mingw32 flags refer to 64 bit windows target
+    export CC=/usr/bin/x86_64-w64-mingw32-gcc
+    export CXX=/usr/bin/x86_64-w64-mingw32-g++
+    export AS=x86_64-w64-mingw32-as
+    export AR=x86_64-w64-mingw32-ar
+    export RANLIB=x86_64-w64-mingw32-ranlib
+    export LD=x86_64-w64-mingw32-ld
+    export STRIP=x86_64-w64-mingw32-strip
+    FFMPEG_OPTS += --target-os=mingw32 \
+                   --arch=x86 \
+                   --cpu=x86-64 \
+                   --cross-prefix=x86_64-w64-mingw32- \
+                   --enable-cross-compile \
+                   --pkg-config=$(shell pwd)/etc/pkg-config-wrapper.sh
```

Comment:
This hack was the only way I found to reliably inject PKG_CONFIG_PATH when configuring FFmpeg for cross-compiling.

The FFmpeg `configure` script overwrites `PKG_CONFIG_PATH` and `PKG_CONFIG_LIBDIR`